src/vtbackend/Screen.h

```diff
@@ -669,7 +669,14 @@ template <CellConcept Cell>
 inline bool Screen<Cell>::isContiguousToCurrentLine(std::string_view continuationChars) const noexcept
 {
     auto const& line = currentLine();
+#if defined(_MSC_VER)
```

Comment:
Put the line variable below to avoid an unused variable

Reply:
Fixed, thanks